python/tvm/auto_scheduler/loop_state.py

```diff
@@ -351,6 +351,72 @@ def compute_root(self, stage):
         self.state_object = _ffi_api.StateComputeRoot(self.state_object,
                                                       self._resolve_stage_id(stage))
 
+    def cache_read(self, stage, scope_name, reader_stages):
+        """ Schedule primitive corresponds to te.schedule.cache_read.
+
```

Comment:
This step does the same thing as `te.schedule.cache_read` does. We choose to not refer to `te.schedule.cache_read` instead of copying the doc string from it.

Reply:
Maybe we can make the pointer more clear. (e.g., say "see also `te.schedule.cache_read`")

Reply:
Add a pointer to `te.schedule.cache_read`, we may also add this to other steps later.